movie/src/hooks/useIsOnMobile.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { useEffect } from "react";
+
+const getIsSamsung = () => window.innerWidth <= 1080;
+const getIsHumbergerIcon = () => window.innerWidth <=1080;
+export default function useIsOnSamsung() {
+    const [isSamsung, setIsSamsung] = useState(getIsSamsung());
```

Comment:
Please find a better naming for the function in this `hooks`

why can we say `isMobile` || `isOnMobile`? why does it need to be `isSamsung` specifically?